src/jit/rationalize.cpp

```diff
@@ -983,6 +983,11 @@ Compiler::fgWalkResult Rationalizer::RewriteNode(GenTree** useEdge, ArrayStack<G
             // Clear the GTF_CALL flag for all nodes but calls
             node->gtFlags &= ~GTF_CALL;
         }
+
+        if (node->TypeGet() == TYP_LONG)
+        {
+            comp->compLongUsed = true;
```

Comment:
Yup. It's set in the importer and lclVar initialization and used in `rpPredictRegUse`.